src/create_project.rs

```diff
@@ -0,0 +1,117 @@
+//! Create a new conductor project
+
+use std::env;
+use std::fs;
+use std::fs::File;
+use std::io::Write;
+use std::path::{PathBuf, Path};
+use util::{Error};
+
+/// Create a new conductor project skeleton
+///
+/// <name>
+/// └── pods
+///   ├── common.env
+///   ├── <name>.yml
+///   └── overrides
+///       ├── development
+///       │   └── common.env
+///       ├── production
+///       │   ├── common.env
+///       └── test
+///           └── common.env
+pub fn create_project(name: &str) -> Result<PathBuf, Error> {
+    let mut cwd = try!(env::current_dir());
+    cwd.push(name);
+
+    try!(create_dir(&cwd));
+
+    let pods: PathBuf = cwd.join("pods");
+    try!(create_dir(&pods));
+
+    try!(write_file(&COMMON_ENV, &pods.join("common.env").as_path()));
+    try!(write_file(&MAIN_YML, &pods.join(format!("{}.yml", name)).as_path()));
+
+    let overrides: PathBuf = pods.join("overrides");
+    try!(create_dir(&overrides));
+
+    for env in ENVIRONMENTS {
+        let dir = overrides.join(env);
+        try!(create_dir(&dir));
+
+        try!(write_file(
+            &format!("DATABASE_URL=postgres://postgres@db:5432/{}_{}", name, env),
```

Comment:
Do we maybe want to output `RAILS_ENV`, `RACK_ENV`  or `NODE_ENV` for each override?